src/server/index.js

```diff
@@ -1,66 +1,78 @@
 import express from 'express';
 import path from 'path';
 import config from 'config';
-import reactRoutes from './react-routes';
 import apiRoutes from './routes/api-routes';
 import bodyParser from 'body-parser';
 import {initMock} from './database';
+import logger from 'winston';
+
 export default (cb) => {
   const app = express();
-  // Please remove it when it's realsed
+  // Please remove it before you release.
   initMock().then(()=>{
-    /* eslint-disable no-console */
-    console.log('Mock data was successfully stored.');
-    /* eslint-enable */
+    logger.info('Mock data was successfully stored.');
+  });
+
+  /*
+    config winston logger
+    @TODO it would be better if you configure winston with log monitoring service(eg. cloudwatch)
+  */
+  const logFileName = `created-logfile.log`;
```

Comment:
how about make directory for log and named log file with date?
like
`log/${new Date().toISOString()}.log`